deploy/operator/setup_assisted_operator.sh

```diff
@@ -180,6 +186,12 @@ spec:
 $(mirror_config)
 EOCR
 
+  if [ "${DISCONNECTED}" = "true" ]; then
+    echo "Adding osImages to AgentServiceConfig because we're in disconnected mode"
+    timeout 10m bash -c "while true; do oc get -n ${ASSISTED_NAMESPACE} agentserviceconfig agent |& grep -ivE '(no resources found|not found)' && break || sleep 10; done"
+    oc patch -n ${ASSISTED_NAMESPACE} agentserviceconfig agent --type merge -p '{"spec":{"osImages":'$(echo "${OS_IMAGES}"| jq -c .|sed 's/openshift_version/openshiftVersion/g; s/cpu_architecture/cpuArchitecture/g; s/rootfs_url/rootFSUrl/g' )'}}'
```

Comment:
those two lines are too complex I'm afraid
we need some way to make it more readable (even if longer in lines)